CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # OHHTTPStubs — CHANGELOG
 
+## [4.4.0](https://github.com/AliSoftware/OHHTTPStubs/releases/tag/4.4.0)
+
+* Added two methods for creating `OHHTTPStubsResponse`s from `NSURL`s that represent file system resources ([@MaxGabriel](https://github.com/MaxGabriel), [#129](https://github.com/AliSoftware/OHHTTPStubs/pull/129))
```

Comment:
Probably just say `added methods for…` (you actually added more than two :wink:)